components/alert/w-alert.vue

```diff
@@ -1,5 +1,41 @@
+<script setup>
+import { computed } from 'vue';
+import { wExpandTransition } from '#generics';
+import { createModel, modelProps } from 'create-v-model';
+import { alert } from '@warp-ds/component-classes';
+
+const props = defineProps({
+  title: String,
+  role: {
+    type: String,
+    default: 'alert'
+  },
+  negative: Boolean,
+  positive: Boolean,
+  warning: Boolean,
+  info: Boolean,
+  ...modelProps(),
+});
+
+const possibleTypeProps = ['negative', 'positive', 'warning', 'info'];
+
+const emit = defineEmits(['update:modelValue']);
+const model = createModel({ props, emit });
+const alertColorType = computed(() => possibleTypeProps.find(e => props[e]));
+const activeWrapperClassNames = computed(() => alert[alertColorType.value]);
+const activeIconClassNames = computed(() => alert[`${alertColorType.value}Icon`]);
+const wrapperClass = computed(() => [
+  alert.alert,
+  activeWrapperClassNames.value
+]);
+const iconClass = computed(() => [
+  alert.icon,
+  activeIconClassNames.value
+]);
+</script>
+
 <template>
-  <div class="f-expandable">
```

Comment:
I found that [f-expandable](https://github.com/fabric-ds/css/blob/next/src/components/expandable.css) class sets `-webkit-tap-highlight-color: transparent;` on the element. Not sure it's still needed, as it's a [non-standard CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color), but should we maybe check if it's safe to remove it?

Reply:
> It would be really helpful if a PR had just the changes that are relevant based on the title/description. Refactoring or changes in other parts of the code base increase "noise" and when it's a new syntax for one's eyes (like vue's is for mine), it extends the review time a bit 😊

I think it's ok at this point to fix small things when we see them, just to progress faster...? But I agree with you that's better... at least when we are at the point of a more "stable" product. 😄 

Anyways I should have added a description so that it was clearer. 👍🙂 
(I am used to getting all the commit messages automatically in the description as default... that would have been handy here. 😄)